app/javascript/packages/webauthn/is-webauthn-platform-authenticator-available.ts

```diff
@@ -0,0 +1,6 @@
+export type isWebauthnPlatformAvailable = () => Promise<boolean>;
+
+const isWebauthnPlatformAuthenticatorAvailable: isWebauthnPlatformAvailable = async () =>
```

Comment:
The [naming convention for TypeScript types would be uppercase](https://github.com/18F/identity-idp/blob/main/docs/frontend.md#naming-conventions), but I also think it'd be fine to inline here?

```suggestion
const isWebauthnPlatformAuthenticatorAvailable = async (): Promise<boolean> =>
```